### PR TITLE
refactor(cli): Migrating from mow.cli to cobra

### LINKED ISSUE #288: Migrating from mow.cli to cobra
Currently we use [mow.cli](https://github.com/jawher/mow.cli) for CLI framework
[Cobra](https://github.com/spf13/cobra) is another popular CLI framework that has these features:

- Intelligent suggestions (app srver... did you mean app server?)
- Automatically generated shell autocomplete for your application (bash, zsh, fish, powershell)
- Automatically generated man pages for your application

It's proposed we move to cobra framework.

### PR DESCRIPTION
**Pull request by kehiy**
## Description

I replaced mow.cli with cobra for wallet commands.

## Related issue(s)

Fixes #288

### CODE REVIEW

**Comment by kehiy** [comment]
@b00f I have no idea about Windows build and coverage, can you review them?

File: cmd/wallet/address.go [hunk 1]
```diff
@@ -3,206 +3,226 @@ package main
...
+	"github.com/spf13/cobra"
 )
 
+var addrCmd = &cobra.Command{
+	Use:   "address",
+	Short: "Manage address book",
+}
+
+func buildAddrCmd(parentCmd *cobra.Command) {
+	parentCmd.AddCommand(addrCmd)
```
Comment (themantre) [inline cmd/wallet/address.go:18]: You can move `addrCmd` inside this function and add all sub commands here:

```go
   buildAllAddressesCmd(addrCmd)
	buildNewAddressCmd(addrCmd)
	buildBalanceCmd(addrCmd)
	buildPrivateKeyCmd(addrCmd)
	buildPublicKeyCmd(addrCmd)
	buildImportPrivateKeyCmd(addrCmd)
	buildSetLabelCmd(addrCmd)
```

File: cmd/wallet/tx.go [hunk 2]
```diff
@@ -1,220 +1,199 @@
...
-				wallet.OptionSequence(int32(*seqOpt)),
-				wallet.OptionMemo(*memoOpt),
-			}
+var txCmd = &cobra.Command{
+	Use:   "tx",
+	Short: "Create, sign and publish a transaction",
+}
+
+func buildTransactionCmd(parentCmd *cobra.Command) {
+	parentCmd.AddCommand(txCmd)
```
Comment (themantre) [inline cmd/wallet/tx.go:17]: Similar to here

File: go.mod [hunk 3]
```diff
@@ -1,16 +1,14 @@
 module github.com/pactus-project/pactus
 
-go 1.19
+go 1.21
```
Comment (themantre) [inline go.mod:3]:
```suggestion
go 1.20
```

File: go.mod [hunk 4]
```diff
@@ -20,17 +18,22 @@ require (
...
 	github.com/tyler-smith/go-bip39 v1.1.0
 	go.nanomsg.org/mangos/v3 v3.4.2
 	golang.org/x/crypto v0.7.0
-	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
 	google.golang.org/grpc v1.53.0
 	google.golang.org/protobuf v1.30.0
 )
 
+require (
+	github.com/hashicorp/golang-lru v0.6.0 // indirect
```
Comment (themantre) [inline go.mod:31]: merge these two blocks.

**Review by themantre — CHANGES_REQUESTED** [review]

**Comment by kehiy** [comment]
@themantre can you have a review?

File: cmd/wallet/address.go [hunk 5]
```diff
@@ -3,206 +3,233 @@ package main
...
 	"fmt"
 
-	cli "github.com/jawher/mow.cli"
 	"github.com/pactus-project/pactus/cmd"
 	"github.com/pactus-project/pactus/crypto/bls"
 	"github.com/pactus-project/pactus/util"
+	"github.com/spf13/cobra"
 )
 
+var addrCmd = &cobra.Command{
```
Comment (themantre) [inline cmd/wallet/address.go:12]: You can move this car inside the `buildAllAddrCmd` (without var)

**Review by themantre — APPROVED** [review]

File: cmd/wallet/address.go (hunk 5 above)
Comment (kehiy) [reply cmd/wallet/address.go:12]: just this?
or I should move all Cmd to their build func?

**Review by b00f — APPROVED** [review]

**Pull request merged**